rest-service/manager_rest/rest/resources_v3_1/license.py

```diff
@@ -38,6 +45,33 @@ def get(self, license_handler, pagination=None):
         return license_handler.list_license()
 
 
+class LicenseCommunity(SecuredResource):
+    @marshal_with(responses_v3.License)
+    def post(self):
+        """
+        Store the Customer ID recieved from Hubspot in the Manager.
+        """
+        request_dict = get_json_and_verify_params({
+            'customer_id': {'type': text_type},
+        })
+        sm = get_storage_manager()
+        if len(sm.list(models.License)) > 0:
+            raise ConflictError(
+                'A Customer ID already exists for this manager')
```

Comment:
Maybe more detailed error message would be useful, e.g. what is the customer_id or something like that?